test/Sentry.Tunnel.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+
+// This file is used by Code Analysis to maintain SuppressMessage 
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given 
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "Test project")]
```

Comment:
I think all test projects should have this rule applied.

Reply:
done